java/src/main/java/io/ably/lib/realtime/AblyRealtime.java

```diff
@@ -0,0 +1,34 @@
+package io.ably.lib.realtime;
+
+import io.ably.lib.platform.JavaPlatform;
+import io.ably.lib.push.Push;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.ChannelOptions;
+import io.ably.lib.types.ClientOptions;
+import io.ably.lib.util.JavaPlatformAgentProvider;
+
+public class AblyRealtime extends AblyRealtimeBase<Push, JavaPlatform, Channel> {
```

Comment:
That's because this class is from the `java` module :wink: Its counterpart in the `android` module is using the `AndroidPlatform` so everything should be fine :ok_hand: